Derivs/src/derivs.cxx

```diff
@@ -87,11 +104,51 @@ calc_derivs(const GF3D5<T> &gf, const vec<GF3D5<T>, dim> &dgf,
         });
     break;
 
+  case 8:
+    grid.loop_int_device<CI, CJ, CK, vsize>(
+        grid.nghostzones,
+        [=] CCTK_DEVICE(const PointDesc &p) CCTK_ATTRIBUTE_ALWAYS_INLINE {
+          const vbool mask = mask_for_loop_tail<vbool>(p.i, p.imax);
+          // Take account of ghost points
+          const vbool mask1 =
+              mask_for_loop_tail<vbool>(p.i, p.imax + deriv_order / 2);
```

Comment:
Better to write `8` instead of `deriv_order`. `8` is a constant, `deriv_order` is not.